frame/lending/src/lib.rs

```diff
@@ -1097,6 +1086,30 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Check is price actual yet
+		fn ensure_price(market: &MarketConfigOf<T>) -> Result<(), DispatchError> {
+			use sp_runtime::traits::CheckedSub as _;
+
+			let borrow_asset = T::Vault::asset_id(&market.borrow_asset_vault)?;
+
+			let current_block = frame_system::Pallet::<T>::block_number();
+			let blocks_count = market.actual_blocks_count;
+			let edge_block = current_block.checked_sub(&blocks_count).unwrap_or_default();
+
+			// check borrow asset
+			let price_block =
+				<T::Oracle as Oracle>::get_price(borrow_asset, Default::default())?.block;
```

Comment:
Perhaps use `ActualType::default()` instead of `Default::default()`, it's much clearer to read

Reply:
changed to `BorrowAmountOf::<Self>::default()`